main.tex

```diff
@@ -424,19 +425,19 @@ \subsection*{Implementing the PROV-DM ontology: WebProv}
 Furthermore, the backend allows one to load in a set of nodes and relationships from JSON into Neo4j on startup to initialize the database.
 
 
-The tool can be locally installed for testing and replicability purposes.
-Details about its installation, as well as the code are found on \href{https://github.com/SFB-ELAINE/WebProv}{GitHub}.
-An informative video showing the usage of \webprov{} is found on \href{https://youtu.be/UzwHtptkYOU}{YouTube}.
+The tool can also be installed locally for testing and replicability purposes.
+Details about its installation, as well as the code, can be found on \href{https://github.com/SFB-ELAINE/WebProv}{GitHub}.
+An informative video showing the usage of \webprov{} is also on \href{https://youtu.be/UzwHtptkYOU}{YouTube}.
 
 
 \subsubsection*{Provenance nodes}
 
 % How is the information of the provenance graph saved in the Neo4j database?
 The main concept of \webprov{} is the Neo4j \provnode{} and the dependency graph created from related \provnodes{} using Neo4j relationships.
 A \provnode{} represents an entity or activity and, therefore, must have a classification (e.g., \SM{} or \BSM{}) which defines the types of relationships that can be formed with other nodes depending on our PROV ontology.
-For example, the entity \SM{} can only be created by either the activity \BSM{} or \CSM{} (see the \nameref{sec:ResultsAndDiscussion} section and Table~\ref{tab:entitiesandactivities} for all details).
-These classifications can be easily changed and extended if needed.
-Additionally, \texttt{Study} nodes store information about a particular study (a study reference as well as the name of the signaling pathway it is concerned with) and group a set of \provnodes{} together.
+For example, the \SM{} entity can only be created by the \BSM{} activity or a \CSM{} (see the \nameref{sec:ResultsAndDiscussion} section and Table~\ref{tab:entitiesandactivities} for all details).
```

Comment:
Throughout the text, I've moved "entity" to follow the entity name rather than have "entity" before. Before, "entity" appeared both before and after the entity name but I find having it after flows much better.

Reply:
I also did this with "activity" too